app/controllers/application_controller.rb

```diff
@@ -11,6 +12,24 @@ def cors_preflight
 
   private
 
+  rescue_from "Exception" do |exception|
```

Comment:
My concern with this is that is now swallows everything (which I know is kind of the point). Makes debugging a bit more difficult. Also don't want us to get lazy in upstream controllers and not raise the appropriate error based on the use case.